packages/expo-development-client/index.js

```diff
@@ -1,5 +1,11 @@
-import { NativeModules } from 'react-native';
+import { AppRegistry, Platform, NativeModules } from 'react-native';
 
-const { EXDevelopmentClient } = NativeModules;
+import App from './bundle/App';
 
-export default EXDevelopmentClient;
```

Comment:
We don't have any JS API for the development client module yet, but if we were to have something so that the user needs to do `import ExpoDevelopmentClient from 'expo-development-client';` then this would not export the module / it would end up registering a different app to the app registry right? So maybe we want to make a different entrypoint file for the bundle and configure metro to use that instead? 🤔

Reply:
Ohh, you're right. I'll make a different entry point for the exported module ;)

Reply:
done ;)